gemp-lotr/gemp-lotr-async/src/main/web/src/Login/login.ts

```diff
@@ -0,0 +1,74 @@
+import { LogInService } from "./services/login.service";
```

Comment:
this is the TS Port of the login page. Login.service is a partial port of the communication.js file. This is good stuff. We have it testable and modularized so we are more aware of what touches what :)